src/Behat/MinkExtension/ServiceContainer/Driver/Selenium2Factory.php

```diff
@@ -98,7 +98,7 @@ protected function getCapabilitiesNode()
                 ->scalarNode('platform')->defaultValue('ANY')->end()
                 ->scalarNode('browserVersion')->defaultValue('9')->end()
                 ->scalarNode('browser')->defaultValue('firefox')->end()
-                ->scalarNode('ignoreZoomSetting')->defaultValue('false')->end()
+                ->booleanNode('ignoreZoomSetting')->defaultFalse()->end()
```

Comment:
Does usage of `booleanNode` instead of `scalarNode` ensure, that if in `.behat.yml` somebody specified `true` or `false` it will be boolean one in parsed version, not string one?